### ISSUE TITLE
Container "recipes_db_recipes_1" continually restarting

### ISSUE THREAD

**Issue by charles-997** [vabene1111/recipes#372]
### Version
Please provide your current version (can be found on the system page since v0.8.4)
Version: latest tag as of 2021 01 31, 6PM EST.  (Cannot launch the software so accessing the system page is not possible)

### Issue
The recipes_db_recipes_1 container continually restarts, thus when I navigate to the serverip:80 in my browser, I get a Server Error (500) page.

### `.env`
Please include your `.env` config file (**make sure to remove/replace all secrets**)
```
# only set this to true when testing/debugging
# when unset: 1 (true) - dont unset this, just for development
DEBUG=0

# hosts the application can run under e.g. recipes.mydomain.com,cooking.mydomain.com,...
ALLOWED_HOSTS=*

# random secret key, use for example base64 /dev/urandom | head -c50 to generate one
SECRET_KEY=***************

# your default timezone
TIMEZONE=America/New_York

# add only a database password if you want to run with the default postgres, otherwise change settings accordingly
DB_ENGINE=django.db.backends.postgresql
POSTGRES_HOST=db_recipes
POSTGRES_PORT=5432
POSTGRES_USER=djangodb
POSTGRES_PASSWORD=******************
POSTGRES_DB=djangodb

# the default value for the user preference 'fractions' (enable/disable fraction support)
# when unset: 0 (disabled)
FRACTION_PREF_DEFAULT=0

# the default value for the user preference 'comments' (enable/disable commenting system)
# when unset: 1 (true)
COMMENT_PREF_DEFAULT=1

# Users can set a amount of time after which the shopping list is refreshed when they are in viewing mode
# This is the minimum interval users can set. Setting this to low will allow users to refresh very frequently which
# might cause high load on the server. (Technically they can obviously refresh as often as they want with their own scripts)
SHOPPING_MIN_AUTOSYNC_INTERVAL=5

# Default for user setting sticky navbar
#STICKY_NAV_PREF_DEFAULT=1

# If staticfiles are stored at a different location uncomment and change accordingly
# STATIC_URL=/static/

# If mediafiles are stored at a different location uncomment and change accordingly
# MEDIA_URL=/media/

# Serve mediafiles directly using gunicorn. Basically everyone recommends not doing this. Please use any of the examples
# provided that include an additional nxginx container to handle media file serving.
# If you know what you are doing turn this back on (1) to serve media files using djangos serve() method.
# when unset: 1 (true) - this is temporary until an appropriate amount of time has passed for everyone to migrate
GUNICORN_MEDIA=0

# allow authentication via reverse proxy (e.g. authelia), leave of if you dont know what you are doing
# see docs for more information https://vabene1111.github.io/recipes/features/authentication/
# when unset: 0 (false)
REVERSE_PROXY_AUTH=0


# allows you to setup o auth providers
# see docs for more information https://vabene1111.github.io/recipes/features/authentication/
# SOCIAL_PROVIDERS = allauth.socialaccount.providers.github, allauth.socialaccount.providers.nextcloud,
```

### `docker-compose.yml`
When running with docker compose please provide your `docker-compose.yml`
```
version: "3"
services:
  db_recipes:
    restart: always
    image: postgres:11-alpine
    volumes:
      - ./postgresql:/var/lib/postgresql/data
    env_file:
      - ./.env

  web_recipes:
    image: vabene1111/recipes
    restart: always
    env_file:
      - ./.env
    volumes:
      - staticfiles:/opt/recipes/staticfiles
      - nginx_config:/opt/recipes/nginx/conf.d
      - ./mediafiles:/opt/recipes/mediafiles
    depends_on:
      - db_recipes

  nginx_recipes:
    image: nginx:mainline-alpine
    restart: always
    ports:
      - 8000:80
    env_file:
      - ./.env
    depends_on:
      - web_recipes
    volumes:
      - nginx_config:/etc/nginx/conf.d:ro
      - staticfiles:/static
      - ./mediafiles:/media

volumes:
  nginx_config:
  staticfiles:
```

I believe this is a setup error, however it is possible that this is a bug.
Thanks in advance to anyone who may be able to point me in the right direction.

**Comment by charles-997** [comment]
UPDATE: (15 min later...)

Based on docker logs it looks like a permissions error within the bind volume `./postgresql`
I tried setting the user to the same UID & GID that my Ubuntu VM uses to access the NFS share on the NAS and that did not solve it.  So instead I have tried using a named volume instead of using a bind volume on the NAS and the error is gone.  Unfortunately this means my postgresql folder is not sitting on my NAS but is instead on the VM's vdisk.

I will close the issue and leave it up in case this is of any help to another user in the future.
